### PR TITLE
[GMCP] add full text to Room.Info gmcp module

### PR DESCRIPTION
**Pull request by lorecrafting**
Added a field to Room.Info GMCP module that has formatted and tagged full text of a room and room's contents.

### CODE REVIEW

File: lib/game/session/gmcp.ex [hunk 1]
```diff
@@ -312,9 +312,10 @@ defmodule Game.Session.GMCP do
 
   defp room_info(room, items) do
     room
-    |> Map.take([:id, :name, :description, :ecology, :x, :y, :map_layer])
+    |> Map.take([:id, :name, :ecology, :x, :y, :map_layer])
     |> Map.merge(%{
       zone: zone_info(room),
+      description: Game.Format.Rooms.room_description(room),
```
Comment (oestrich) [inline lib/game/session/gmcp.ex]: Let's do an alias at the top for `Game.Format`.

File: lib/game/session/gmcp.ex [hunk 2]
```diff
@@ -394,6 +395,12 @@ defmodule Game.Session.GMCP do
       %{^key => data} when data != nil ->
         render_many(data)
 
+      %{:npcs => npcs} when npcs != nil ->
```
Comment (oestrich) [inline lib/game/session/gmcp.ex]: For these I would pattern match on the `key` variable. So above these lines, do something similar to the `:exits` above.

```elixir
def render_many(npcs, :npcs) do
   # ...
end
```

File: lib/game/session/gmcp.ex (hunk 1 above)
Comment (lorecrafting) [reply lib/game/session/gmcp.ex]: ✅

File: lib/game/session/gmcp.ex (hunk 2 above)
Comment (lorecrafting) [reply lib/game/session/gmcp.ex]: ✅

**Pull request merged**